public/scripts/acquisition.js

```diff
@@ -118,9 +109,10 @@ function enterSession(key) {
                 'uConnected': uConnected,
                 'timestamp': Date.now()
             });
-
 		
-
+            //The issue #5 may also be occuring here. Some questions to ask include
+            //"is this lambda actually being triggered?" and "Does the .ref() actually invoke an update?"
+            //Why would this be invoked in the "onKeyEnter" function?
```

Comment:
you know what, this might be why this is not running as expected. I'll try adding an eventlistener for window here and attach the lambda to that instead. #5

Reply:
I'll let you take a peek at that!

Reply:
I'll let you take a peek at that!